week-4/Homework/mandatory/3-water-bottle.js

```diff
@@ -18,23 +18,44 @@ You have to implement the missing features according to the specification.
 */
 
 // Here is your starting point:
+// calling this function should pour your bottle full (volume = 100);
 let bottle = {
   volume: 0,
-  fillUp: function() {
-    // calling this function should pour your bottle full (volume = 100);
+  fillUp: function () {
+    this.volume += 100;
   },
+  // calling this function should increase your bottle volume by 10 unit;
   pour: function () {
-    // calling this function should increase your bottle volume by 10 unit;
+    if (this.volume <= 90) {
+      this.volume += 10;
+    }
   },

```

Comment:
Read carefully the instruction regarding this part Leida.
pouring 10 unit of water into it
    Note: You cannot over exceed the bottle capacity.

Reply:
I did not understand this one Abdoulrazak, I used the condition if (this.volume <= 90) exactly to ensure it does not go over 100 units. Any hints on what exactly I am getting wrong so I can fix it?